pkg/operator/main.go

```diff
@@ -60,6 +60,7 @@ type Config struct {
 	LogLevel                               string `env:"LOG_LEVEL" envDefault:"info"`
 	MinPort                                int32  `env:"MIN_PORT" envDefault:"10000"`
 	MaxPort                                int32  `env:"MAX_PORT" envDefault:"12000"`
+	ListeningPort                          int32  `env:"LISTENING_PORT" envDefault:"5000"`
```

Comment:
like in the env variable name, let's call the `ListeningPort` with a more descriptive name, like `AllocationApiServicePort` or something

Reply:
Adjusted the name of the var in all services